clang/docs/LanguageExtensions.rst

```diff
@@ -4514,9 +4514,13 @@ default member initializer, the invocation point is the location of the
 constructor or aggregate initialization used to create the object. Otherwise
 the invocation point is the same as the location of the builtin.
 
-When the invocation point of ``__builtin_FUNCTION`` is not a function scope the
+When the invocation point of ``__builtin_FUNCTION`` is not a function scope, the
 empty string is returned.
 
+The builtin ``__builtin_COLUMN`` returns the offset from the start of the line,
+beginning from column 1. `This may differ from other implementations.
+<https://en.cppreference.com/w/cpp/utility/source_location/column>`_
+
```

Comment:
You could link to the standard instead of cppreference https://eel.is/c++draft/support.srcloc#tab:support.srcloc.current-row-3-column-2-sentence-2